pkg/controller/humiocluster/humiocluster_controller.go

```diff
@@ -1381,8 +1470,13 @@ func (r *ReconcileHumioCluster) ensurePodsExist(ctx context.Context, hc *corev1a
 		return reconcile.Result{}, err
 	}
 
+	attachments, err := r.newPodAttachments(ctx, hc, foundPodList)
+	if err != nil {
+		r.logger.Errorf("failed to get pod attachments: %s", err)
```

Comment:
@jswoods I think we might be missing a return statement here. I assume we'd want to return the error?

Reply:
fixed